crates/proto/src/lib.rs

```diff
@@ -4,5 +4,12 @@ use serde::{Deserialize, Serialize};
 pub struct Rustacean {
     pub name: String,
     pub image: String,
+    pub bio:String,
     pub github_url: String,
```

Comment:
Not part of this PR but given that we rely on GitHub for identity validation, I would just leave this either as `id` or `gh_user` (as it is a bit more compact)